modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -7,29 +7,73 @@ endif::[]
+
+If the person with whom you are working uses Anypoint Studio, make the JAR file available to that person, provide them with all of the connectivity information that they need for running the app, and then ask them to follow these steps:
+
+. Enable Studio to access your organization in Anypoint Platform by selecting *Preferences* > *Anypoint Studio* > *Authentication*.
+. Import the JAR file into Studio by selecting *File* > *Import* and then selecting *Anypoint Studio* > *Packaged Mule Application (.jar)*.
++
+Studio opens the project.
+
+. Open the file `src/main/resources/mule-artifact.properties` in the project and provide values for the keys that are related to connection information.
+
+. Save `src/main/resources/mule-artifact.properties`.
+
+== Steps to Follow in Flow Designer to Import a Project
+
+If the person with whom you are working is in a different organization in Anypoint Platform and uses Flow Designer, make the JAR file available to that person, provide them with all of the connectivity information that they need for running the app, and then ask them to follow these steps:
+
+. Create a new Mule-app project in Flow Designer by opening Design Center, clicking *Create New*, and then clicking *Import Application* under *Create a Mule Application*.
+. From the *Import Application* dialog, select the JAR file and then click *Import*.
+. Provide the connection information in the configurations that the connectors in the project use.
++
+When Flow Designer opens the project, each card that represents a connector displays a red circle in its top-right corner. You must add the connection information that Flow Designer did not include in the JAR file.
+
+.. In the *Project* pane on the left side of the canvas, expand the section *Reusable Configurations*.
++
+All of the configurations that are used by connectors are marked by a red circle.
++
+image::fd-location-of-red-circles-on-configs.png[]
 +
-. Alert the recipient who works in Studio that the `.jar` is available so that they can:
-.. Access the `.jar` file and place it in a location where they can easily upload it from Studio.
-.. Enable Studio to access your organization in Anypoint Platform by selecting *Preferences* > *Anypoint Studio* > *Authentication*.
-.. Import the `.jar` file by selecting *File* > *Import* and then selecting *Anypoint Studio* > *Packaged Mule Application (.jar)*.
+<1> The icon to click to open the *Project* pane.
+<2> A red circle to indicate that the flow contains errors. Because the configurations in the project lack connection information, there is a red circle next to each flow that includes connectors.
+<3> The *Reusable Configurations* section, which lists the configurations that connectors and core components in a project are using.
+<4> A red circle next to a listed configuration.
+
+.. Click a configuration.
+.. In the dialog that is opened, click the three dots in the connection that is being used in the configuration and select *Edit*.
```

Comment:
Is it necessary to include the opening phrase; does not a next step assume the result of the first?